README.md

```diff
@@ -7,7 +7,34 @@ BE Mod 2 Week 2/3 Pair Project
 
 Students will be put into pairs to complete the project.
 
+## The Contributors:
+
+### Christopher Allbritton
```

Comment:
Thank you so much for adding this breakdown! 

It was such a pleasure to work with you on this.